WalletWasabi.Fluent/Views/CoinControl/Core/Cells/SelectionCellView.axaml

```diff
@@ -0,0 +1,11 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:core="clr-namespace:WalletWasabi.Fluent.ViewModels.CoinControl.Core"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:Class="WalletWasabi.Fluent.Views.CoinControl.Core.Cells.SelectionCellView"
+             x:DataType="core:CoinControlItemViewModelBase"
+             x:CompileBindings="True">
+  <CheckBox IsChecked="{Binding IsSelected}" IsEnabled="{Binding CanBeSelected}" />
```

Comment:
We agreed in a previous PR that we should rather hide the checkmark completely. https://github.com/zkSNACKs/WalletWasabi/pull/9657#discussion_r1034624855

Reply:
That's correct. Forgot to disable them.